interface/globals.php

```diff
@@ -101,14 +101,10 @@
         $tmp = $_GET['site'];
     } else {
         if (empty($ignoreAuth)) {
-            // mdsupport - Don't die if logout menu link is called from expired session.
-            // Eliminate this code when close method is available for session management.
-            if ((isset($_GET['auth'])) && ($_GET['auth'] == "logout")) {
-                $GLOBALS['login_screen'] = "login_screen.php";
-                $srcdir = "../library";
-                include_once("$srcdir/auth.inc");
-            }
-            die("Site ID is missing from session data!");
+            $GLOBALS['login_screen'] = "login_screen.php";
+            $srcdir = $web_root."/library";
+            include_once("$srcdir/auth.inc");
+            exit;
```

Comment:
@mdsupport, since you've worked on this, do you see any issues with this code?

Reply:
Sorry, missed this.

@ophthal : 
This code can get called in various contexts (think old frames) .  We limited the scope in current code to a specific instance when user specifically is trying to logout from expired session.  That input is recognized by GET and (hopefully) it would come from main menu.

We tried it without the restriction but it led to strange situations - e.g. login screen in multiple frames/iframes.  For users it got confusing.

There is real need to avoid extremely irritating site ID message that every user needs an explanation!  Solution for tabs interface is to call a location function in the **top** window's document. Let that code die if somehow that function is missing - that tells you did not reach correct **top**.